gitpod-io/gitpod
# [db] Implement new Team and Project entities

## How to test

- (I'm not sure how to run the unit test)

- Log in to https://jx-teams-and-projects.staging.gitpod-dev.com/ and check the Console (it should log `Teams: []`)

- Connect to the DB, manually create a new team, and assign yourself to it:

```
mysql> insert into d_b_team (id, name, creationDate) values (UUID(), 'YOUR TEAM', '2021-06-03T10:00:00.000Z');

mysql> select * from d_b_team;

mysql> select id, name from d_b_user;

mysql> insert into d_b_team_membership (id, teamId, userId) values (UUID(), 'YOUR-TEAM-ID', 'YOUR-USER-ID');
```

- Refresh the dashboard -- the console should now log `Teams: [{ ... }]` (i.e. your team)
